tests/e2e/vsphere_volume_expansion.go

```diff
@@ -651,6 +651,8 @@ var _ = ginkgo.Describe("Volume Expansion Test", func() {
 		var originalSizeInMb, fsSize int64
 		var err error
 		var expectedErrMsg string
+		vcAddress := e2eVSphere.Config.Global.VCenterHostname + ":" + sshdPort
+		featureEnabled := isFssEnabled(vcAddress, "CNS_NEW_SYNC")
 
 		volHandle, pvclaim, pv, storageclass := createSCwithVolumeExpansionTrueAndDynamicPVC(
 			f, client, "", storagePolicyName, namespace)
```

Comment:
In defer block , Line number 660  and 1446 , We should not delete storage class in case of supervisor cluster